test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -264,5 +261,29 @@ public void CanOpenDocumentationBrowserWhenMoreInformationIsClicked()
             DispatcherUtil.DoEvents();
         }
 
+        /// <summary>
+        /// This test checks that the IronPython dialog is shown to the user,
+        /// when the workspace has custom nodes that contain a python node in it. 
+        /// </summary>
+        [Test]
+        public void WillDisplayDialogWhenCustomNodeInsideWorkspaceHasIronPythonNode()
+        {
+            DebugModes.LoadDebugModesStatusFromConfig(Path.Combine(GetTestDirectory(ExecutingDirectory), "DynamoCoreWpfTests", "python2ObsoleteMode.config"));
+            DynamoModel.IsTestMode = false;
```

Comment:
why do we need to set test mode false?

Reply:
In the test mode true, we don't show the dialogs. https://github.com/DynamoDS/Dynamo/blob/master/src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs#L139